analyzer/topology_server.go

```diff
@@ -275,30 +280,31 @@ func (t *TopologyServer) OnWSJSONMessage(c shttp.WSSpeaker, msg *shttp.WSJSONMes
 	// this kind of message usually comes from external clients like the WebUI
 	if msgType == graph.SyncRequestMsgType {
 		t.Graph.RLock()
+		defer t.Graph.RUnlock()
```

Comment:
I would prefer to unlock the graph before sending the message so that we don't keep the graph lock if the send queue is full (do not use defer and do t.Graph.RUnlock() before the 2 "return" statements

Reply:
Ok, Did change.